SwiftCompilerSources/Sources/AST/SubstitutionMap.swift

```diff
@@ -28,6 +28,15 @@ public struct SubstitutionMap: CustomStringConvertible {
   public init() {
     self.bridged = BridgedSubstitutionMap()
   }
+  
+  public init(forGenericSignatureOf functionType: CanonicalType,
```

Comment:
This should just take a generic signature and not a generic function type.

Reply:
We don't have GenericSignature bridged to Swift yet. I can do this in a follow up PR